apiary.apib

```diff
@@ -3250,6 +3250,357 @@ Update the current timer. Only possible if there is a timer running.
 
 + Response 204
 
+# Group Work Orders
+
+## Work Orders [/workOrders]
+
+### workOrders.draft [POST /workOrders.draft]
+
+Add a draft work order.
+
++ Request (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + customer (object, required)
+            + type (enum[string], required)
+                + Members
+                    + contact
+                    + company
+            + id: `09e5d75f-f817-4872-9723-57fbb8ff710d` (string, required)
+        + performed_at: `2017-05-26T10:01:49+00:00` (string, required)
+        + description: `Pipework intervention` (string, optional)
```

Comment:
Q: can we keep this optional when we're starting to build an overview of workorders somewhere? I guess we'll need this to show which workorder is which?

Reply:
Good question. I think this will have to be checked with UX and the Mobile team. Anyway the description is optional in the current client API.

@JorenSaey do you have wireframes and if so what is specified on the WorkOrder overview?

Reply:
I think we can do the same thing like TimeTracking: display it like (no description) in the ListView. If it's optional now, the next api should perhaps also make it optional. My opinion is that it is not for front end to decide whether something is required or not. We have other ways to show this ;)

Reply:
To be checked. 👍